.github/actions/run-bwc-suite/action.yaml

```diff
@@ -0,0 +1,46 @@
+name: 'Create a backwards compatible ready build'
+description: 'Checkouts the official version of a the Security plugin and builds it so it can be used for BWC tests'
+
+inputs:
+  plugin-previous-branch:
+    description: 'The branch of the plugin that should be built for the previous version, e.g "2.2", "1.x"'
+    required: true
+
+  plugin-next-branch:
+    description: 'The branch of the plugin that should be built for the next version, e.g "2.3", "main"'
```

Comment:
Can we just increment automatically by saying 2.n --> 2.n+1 and  1.x --> 2.0, 2.x --> 3.0? Or is this not quite consistent with our naming conventions?

Reply:
This parameter is meant to allow for different kinds of testing e.g. between to small changes by SHAs, between 1.2 -> 2.2 or between 1.2 -> main.  While these scenarios at not necessary supported, this allows testing them in the event we wanted to test them.

We could describe the version flow pattern and pick up the N+1 version, but I figure that makes more sense out of band of this change.

Reply:
This makes sense then. I just wanted to make sure I understood your plan.